CHANGELOG.md

```diff
@@ -1,3 +1,79 @@
+# v0.23.0
+
+## Announcements
+
+- Re-licensing: license change from
+  [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) to
+  [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0) (see #997 for
+  details)
+- `sbi` is now affiliated with [`NumFOCUS`](https://numfocus.org) 🎉
+- New contributors 🎉: @anastasiakrouglova, @theogruner, @felixp8, @Matthijspals,
+  @jsvetter, @pfuhr, @turnmanh, @fariedabuzaid, @augustes, @zinastef, @Baschdl,
+  @danielmk, @lisahaxel, @janko-petkovic, @samadpls, @ThomasGesseyJonesPX, @schroedk
+
+## Major Changes and New Features
+
+- change `sbi` default parameters: `training_batch_size=200`, `num_chains=20` (#1221,
+  @janfb)
+- more flexible API and shape handling for density estimator classes (#952, #965, #979,
+  #1151) (@gmoss13, @tomMoral, @manualgloeckler)
+- vectorized sampling and log_prob for `(S)NPE` given batches of x (#1153)
+  (@manuelgloeckler, @deismic)
```

Comment:
Add somewhere that `posterior_nn` should be imported from `sbi.neural_nets` now, not from `sbi.utils`